test/PushNotificationsTests/APITests.cpp

```diff
@@ -91,13 +91,65 @@ namespace Test::PushNotifications
             return true;
         }
 
-        TEST_METHOD(LaunchPackagedApp)
+        TEST_METHOD(ChannelRequestUsingNullRemoteId)
```

Comment:
Where is the code calling the APIs? Or are these all placeholders?

Reply:
I thought tests are being checked into the sperate topic branch for testing?